packages/p2p/src/components/empty/empty.scss

```diff
@@ -27,8 +33,12 @@
             line-height: 1.43;
         }
     }
-    &__button {
-        text-decoration: none;
-        margin-top: 1.6rem;
+
+    &-title {
+        color: var(--text-general);
+        font-size: var(--text-size-s);
+        font-weight: bold;
+        line-height: 1.5;
+        margin: 0.8rem;
```

Comment:
Can we use `<Text>` component instead?

Reply:
Yes, we can. [Refer: Bob the Builder Theme Song]